packages/frontend/src/features/scheduler/components/SchedulerForm.tsx

```diff
@@ -343,6 +358,11 @@ const SchedulerForm: FC<Props> = ({
                         <Tabs.Tab value="filters">Filters</Tabs.Tab>
                     ) : null}
                     <Tabs.Tab value="customization">Customization</Tabs.Tab>
+                    {form.values.format === SchedulerFormat.IMAGE && (
+                        <Tabs.Tab value="preview">
+                            Preview and change layout
```

Comment:
This tab title looks a bit long in next to the others. How about just 'Preview'? Or 'Preview and size'?

We should probably chat the Customization tab name too, but that a little out of scope.

Reply:
Agree with `Preview and Size` 👍 Yea, was thinking that as well, should they even be on the same Tab? 🤔